cli/src/main/java/com/mrv/yangtools/codegen/main/Main.java

```diff
@@ -38,7 +38,13 @@
 
     @Argument(multiValued = true, usage = "List of YANG module names to generate in swagger output", metaVar = "module ...")
     List<String> modules;
-
+    
+    @Option(name = "-format", usage = "Output format of generated file - defaults to yaml with options of json or yaml", metaVar = "enum")
+    public SwaggerGenerator.Format outputFormat = SwaggerGenerator.Format.YAML;
+    
+    @Option(name = "-api-version", usage = "Version of api generated - default 1.0", metaVar = "file", )
```

Comment:
I got the following error:
`[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /usr/src/app/cli/src/main/java/com/mrv/yangtools/codegen/main/Main.java:[45,104] illegal start of expression
[INFO] 1 error`
but removing the trailing comma fixed it.